### DISCUSSION TITLE
Open/Connect to db fails witth new version

### DISCUSSION THREAD

**Discussion by Farkled** [pawelsalawa/sqlitestudio#4821] [Q&A]
Try as I might, I cannot get app to open/connect to a db even though my apps read & write the files just fine.  I had no issues before the upgrade.  Pretty sure it is me but I need help - how do I connect Studio to a db?

**Comment by pawelsalawa** [comment]
How does your problem appear exactly? What you're doing, what is expected behavior and what is actual behavior?

**Reply by Farkled** [reply]
Using add a databse from menu brings up the screen where I enter the file name (*.db) and it suggest a list name.  Press test connection and get green check.  Then press OK.  Screen disappears.  Menu option to connect to a db is still greyed out.  I've tried every combination of filename variants I can think of - using back slashes, forward slashes. adding or deleting the ".db" in the filename.  The dbs and tables are there and working fine.  All programs read/write tables just fine.  The SQLManager that runs as a Chrome extension has no problems.

SQLiteStudio is my preferred manager since I finally (had) mastered it sufficiently for my needs - and now I really need it to visualize some data I need to analyze.  Thank you for the quick response.

**Reply by pawelsalawa** [reply]
Did you select which database you would like to connect to? (you know, you can have multiple databases added to the list)

![image](https://github.com/pawelsalawa/sqlitestudio/assets/481846/829e1cac-26a2-402d-a8d9-b1c5e63f69d4)

The connect button will be grayed out until you select a database you wish to connect to.

**Reply by pawelsalawa** [reply]
You can also just double-click on a database on the list to connect to it.

**Reply by Farkled** [reply]
Unable to gete past the 1st screen to add a db - even when selected from the file listing (that it comes up where the data actually resides  is much appreciated.)

**Reply by pawelsalawa** [reply]
I don't understand. You've said, you selected a file and tested the connection, which gave you green status. Then you pressed OK and the db dialog disappeared, which suggests that the database has been added to your list of databases.

Do you have your database list panel displayed? Check the "View" menu and look for "Database list" there.

**Reply by Farkled** [reply]
I do not understand either.  Select from the list and get the error flag.  Change it by hand and the flag goes away.  The connection then tests green.  Click OK and the screen goes away.  The connect button is greyed out as is the connect option in the menu.  There is just no list of dbs.  I think the next step is to uninstall and then do a fresh install

**Reply by pawelsalawa** [reply]
Reinstalling will not help you, if you don't use the app properly.

Do you see this panel?
![image](https://github.com/pawelsalawa/sqlitestudio/assets/481846/1d4e4f2f-f79c-472d-839d-116e9b7c3c75)

If not, can you make sure you have it enabled here?
![image](https://github.com/pawelsalawa/sqlitestudio/assets/481846/96f6908f-5a24-486a-b1ea-d07acfc3b3c0)

**Reply by Farkled** [reply]
Fresh install results in same issues.

**Reply by Farkled** [reply]
Yes, I am trying the Database menu option inn the upper left.  It shows nothing becuayse the add database screen does not add a connection.  Incidentally, the Ctrl-O approach does nothing either - maybe that adds a clue.

if I understand correctly, step one is to add a database -and that is where I am stopped.  That screen worked fine in the prior version.  I cnnot make the error flag in the filename field go away with anything legitimate, but the connection test green.

**Reply by Farkled** [reply]
I am hoping this screen shot will illustrate the issue I am having.
![Screenshot (9)](https://github.com/pawelsalawa/sqlitestudio/assets/118643375/bc94a1be-f48b-48d3-a40b-2373b01d0941)

**Reply by pawelsalawa** [reply]
Okay. My guess is that you already have this database added to your database list. You can confirm it by moving your mouse over this red indicator in top-left corner of file input. The application prevents you from adding the same database twice.

Just use this database from the already added item on the database list.

**Reply by Farkled** [reply]
As the creenshot shows, although the dbs have been added as you stated,there is no way to connect the app to them.  The connect options are greyed out.
![Screenshot (10)](https://github.com/pawelsalawa/sqlitestudio/assets/118643375/e47d6168-56dc-4f39-9203-974151e05c5c)

**Reply by pawelsalawa** [reply]
As we can see on your screenshot, you do NOT have database list panel displayed. I asked you to make sure you have it in this comment above: https://github.com/pawelsalawa/sqlitestudio/discussions/4821#discussioncomment-6338777 (my two posts earlier)

**Reply by Farkled** [reply]
My apologies sir.  I did not see the database line above the bolder selected items list.  The fact that I need to run an inverted color scheme may have been a factor or maybe I'n just not smart enough to be messing with code.  Most likely, I think, is that my vision has failed to the point where I need to run a scene magnifier at 2x or 3x magnification levels to read anything.
I very much appreciate your prompt and outstanding support.  I am relieved because you app is easily the best SQLManager I have seen.  I was struggling without it.
With my heartfelt thanks,  .....

**Reply by pawelsalawa** [reply]
I'm happy I could help. Cheers!
